vaadin-upload-flow-parent/vaadin-upload-flow/src/main/java/com/vaadin/flow/component/upload/Upload.java

```diff
@@ -88,8 +88,12 @@ public Upload() {
         DomEventListener allFinishedListener = e -> {
             JsonArray files = e.getEventData().getArray(elementFiles);
 
-            boolean isUploading = IntStream.range(0, files.length()).anyMatch(
-                    index -> files.getObject(index).getBoolean("uploading"));
+            boolean isUploading = IntStream.range(0, files.length())
+                    .anyMatch(index -> {
+                        final String KEY = "uploading";
+                        JsonObject object = files.getObject(index);
+                        return object.hasKey(KEY) && object.getBoolean(KEY);
```

Comment:
The construct here is a bit unfortunate, but it seems that there is no way to ask Flow to return an optional or a default value? If we need to do this more often, we might want to introduce a helper for that.

Reply:
It is a valid concern, but `JsonObject` is not part of Flow and it does not provide an API using Optional.